README.md

```diff
@@ -1,3 +1,312 @@
-TuxedoJS
-========
+# TuxedoJS
+
 ![Tux Logo](Tux-full-logo.png "Tux-Logo")
+
+
+> A Front-End Javascript Framework built on Facebook’s powerful React view layer and Flux architecture.
+
+`React` and `Flux` are two novel and exciting approaches to front-end development. `TuxedoJS` implements a more accessible and semantic toolkit that leverages these technologies, abstracting away unnecessary complexity and extending core functionality. `TuxedoJS` comes from `Tux` - a merging of `React` and `Flux`.
+
+## Table of Contents
+
+1. [Features](#features-and-examples)
+1. [Graceful Degradation](#graceful-degradation)
+1. [Requirements](#requirements)
+1. [Development](#development)
+    1. [Installing Dependencies](#installing-dependencies)
+    1. [Tasks](#tasks)
+    1. [Roadmap](#roadmap)
+1. [Credits and Tech Stack](#credits-and-tech-stack)
+1. [Contributing](#interested-in-contributing)
+1. [Team](#team)
+
```

Comment:
Can we have a real quick overview of `React` and `Flux` up here and then a quick overview on where `Tux` fits in. Also, in this section, can we link to our website.